### PR TITLE
Updates for arch-based node image

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -24,8 +24,8 @@ function generate_library(){
 	echo '# maintainer: Trong Nghia Nguyen - resin.io <james@resin.io>' >> $lib_name
 
 	for version in "${versions[@]}"; do
-		commit="$(git log -1 --format='format:%H' -- "$path/$version")"
-		fullVersion="$(grep -m1 'ENV NODE_VERSION ' "$path/$version/Dockerfile" | cut -d' ' -f3)"
+		commit="$(git log -1 --format='format:%H' -- "$path/$version/slim")"
```
Comment (petrosagg) [inline generate-stackbrew-library.sh:27]: Can you update the commit message explaining what this change is for?

Comment (nghiant2710) [inline generate-stackbrew-library.sh:27]: @petrosagg done, I've updated the commit message.

**Comment by petrosagg** [comment]
d9de52d16f9db8ff063cb170487671a61fbbd6ce should be split in 3 commits

File: generate-stackbrew-library.sh [hunk 2]
```diff
@@ -55,7 +55,7 @@ function generate_library(){
 
 declare -A aliases
 aliases=(
-	[0.12.7]='0 latest'
+	[5.10.0]='0 latest'
```
Comment (petrosagg) [inline generate-stackbrew-library.sh]: The alias should be `5`, not `0`

Comment (nghiant2710) [inline generate-stackbrew-library.sh]: Updated!

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**